src/case-swapping.js

```diff
@@ -15,7 +16,17 @@
  * @return {string}
  */
 function swapCase(phrase) {
-  // write code here
+  let resultPhrase = '';
+
+  for (const symbol of phrase) {
+    if (symbol === symbol.toLowerCase()) {
+      resultPhrase = resultPhrase + symbol.toUpperCase();
```

Comment:
there is a short entry
```suggestion
      resultPhrase += symbol.toUpperCase();
```